_posts/2021-05-23-irc-move.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: "NewPipe bides farewell to freenode and can now be found on Libera.Chat"
+short: "NewPipe moves to Libera.Chat"
+date: 2021-05-23 09:00:00 + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+We wrote an in-depth explanation below, but if you do not need or care about this, for you in a few words: Newpipe has an active IRC channel where developers and users chat and can ask questions. This channel moves from the freenode to the Libera.Chat network. Until Matrix finalizes its bridge to the new network, the room on freenode will be active, but as soon as that happens, it will be shut down.
+<!-- more -->
+
+## What is IRC?
+
+In order to provide a bit of a background, what follows will be short explanation what all this is. If you are familiar with IRC, skip to the next section.
+
+[freenode](https://freenode.net/) and [Libera.Chat](https://libera.chat) are IRC networks. IRC stands for Internet Relay Chat. Without diving too deep in technical explanations (if you care for this, read the [wikipedia article](https://en.wikipedia.org/wiki/Internet_Relay_Chat)), it is an instant communication standard for groups. You install an IRC app on your device (or use a webapp), those act as IRC clients, which you then need to point to an IRC server. Bigger IRC servers are called networks, because instead of just one server, they are a network of several servers (makes sense). The biggest and oldest IRC network is called freenode, and we as NewPipe used this network to host our channel. Up until this week.
+
+## What has happened?
+
+This will be a very short explanation again. This time, we will condense what apparently has happened behind the scenes of freenode. If you want to paint your own picture, read [this post](https://gist.github.com/realrasengan/88549ec34ee32d01629354e4075d2d48) from one side of this drama, and [this resigning letter](https://gist.github.com/aaronmdjones/1a9a93ded5b7d162c3f58bdd66b8f491) from the other. There are many more resigning letters around, you can search for it.
+
+In a nutshell, about a dozen volunteer developers behind freenode left the project. This happened because Andrew Lee bought freenode Limited in 2017, a company which was used as a legal representation of freenode, mostly for their freenode Live Conferences. While he promised to not obtain any influence over freenode itself, this apparently changed this week. Andrew and his team gained full controll over the network (as stated by their [twitter account](https://twitter.com/freenodestaff/status/1395046345145307140)), which lead to the volunteers resign. Whilste resigning, mosted of the volunteers stated they will now be at Libera.Chat.
```

Comment:
```suggestion
In a nutshell, about a dozen volunteer developers behind freenode left the project recently. This happened because Andrew Lee bought Freenode Limited in 2017, a company which was used as a legal representation of freenode, mostly for their freenode Live Conferences. While he promised not to obtain any influence over freenode itself, this apparently changed this week. Andrew and his team gained full control over the network (as stated by their [Twitter account](https://twitter.com/freenodestaff/status/1395046345145307140)), which led to the volunteers resigning. While resigning, most of the volunteers stated they will now be at Libera.Chat.
```